test/stat/test_chmod.c

```diff
@@ -111,7 +111,7 @@ void test() {
   memset(&s, 0, sizeof s);
   stat("folder", &s);
   assert(s.st_mode == (0300 | S_IFDIR));
-  assert(s.st_ctime != lastctime);
```

Comment:
I think these assertions are valid, no?  When I run this test on native linux they all seem to pass.

Reply:
Oh I see, so linux does track ctime separately, but emscripten doesn't (at least not today).

I wonder if its worth adding separate ctime tracking to emscripten?  Otherwise I guess we should make these assertions platform dependent?

For example if we want to make this test pass with NODERAWFS then we would want these assertions still I think.

Reply:
If we tracked atime mtime and ctime separately, that would allow us to be more posix compliant but take up a little bit more memory per file? Seems to me like it would be worth it since I'd expect the contents of files to dominate the metadata.

Reply:
Yes, I imagine its more about the static codesize/complexity overhead.   Perhaps we can make that change before we make this one?  Or as part of it?

Reply:
I created https://github.com/emscripten-core/emscripten/pull/23004 which runs this test also under noderawfs

Reply:
Okay I added another commit that maintains separate atime, mtime, and ctime. Should we support `O_NOATIME` too?